crates/transaction-pool/src/lib.rs

```diff
@@ -421,7 +421,7 @@ where
         &self,
         base_fee: u64,
     ) -> Box<dyn BestTransactions<Item = Arc<ValidPoolTransaction<Self::Transaction>>>> {
-        self.pool.best_transactions_with_base_fee(base_fee)
+        self.pool.best_transactions_with_attributes(BestTransactionsAttributes::base_fee(base_fee))
```

Comment:
@mattsse I just did an update, this is the only place where I use the base fee initializer. On the other places, I use an initialization taking into account the gas price of the blob and if there is no blob, then the Option will just return a None and it will have the same effect as the base fee initializer in my opinion.

Tell me what you think?